packages/auth/__tests__/auth-unit-test.ts

```diff
@@ -491,7 +491,21 @@ describe('auth unit test', () => {
 			const spyon = jest.spyOn(CognitoUser.prototype, 'confirmRegistration');
 			const auth = new Auth(authOptions);
 
-			expect.assertions(1);
+			const unsubscribe = Hub.listen('auth', capsule => {
+				switch (capsule.payload.event) {
+					case 'confirmSignUp': {
+						unsubscribe();
+						return expect(capsule.payload.data).toBe('Success');
+					}
+
+					case 'confirmSignUp_failure': {
+						unsubscribe();
+						return expect(capsule.payload.data).toBe('false');
```

Comment:
What am I missing here? Should it be as below: 
```suggestion
						return expect(capsule.payload.data).toBe(false);
```

Reply:
This (failure) block never fires, but is here for completeness in the event a mock or something breaks around it. (Otherwise we'll only get a "Expected 2 assertions, got 1" error from the test.)

We could either make it similar to the test below:

```suggestion
						return expect(capsule.payload.data).toBe('Should not be called');
```

Or **remove it entirely**.  Honestly, I'm totally fine to only have the needed `expect` in place with `expect.assertions(2)`!

Reply:
That makes sense to me. I believe we should keep it in case we miss this writing additional test in the future